<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>693</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>831</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="gbRegisters">
      <property name="title">
       <string>Регистры</string>
      </property>
      <widget class="QLineEdit" name="line_in">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>571</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="line_out">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>571</width>
         <height>31</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>69</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Ввод</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>61</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Вывод</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbOperators">
      <property name="title">
       <string>Операторы</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QRadioButton" name="b_X">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Оператор X</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="b_Y">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Оператор Y</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="b_Z">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Оператор Z</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="b_H">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>121</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Оп. Адамара</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="b_CCNOT">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Оп. CCNOT</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="b_R">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>151</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Оп. фаз. сдвига</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="b_Fourier">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>211</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Пр-ние Фурье</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>30</y>
         <width>301</width>
         <height>131</height>
        </rect>
       </property>
       <property name="title">
        <string>Параметры</string>
       </property>
       <widget class="QScrollArea" name="scrollArea">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>301</width>
          <height>111</height>
         </rect>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>299</width>
           <height>109</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QRadioButton" name="b_CNOT">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>100</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Оп. CNOT</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="but_ACT">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
